client/post-editor/post-editor.jsx

```diff
@@ -423,12 +423,7 @@ export const PostEditor = React.createClass( {
 		// Some browsers, as Firefox, return a DOMStringList instead, see
 		// https://developer.mozilla.org/en-US/docs/Web/API/DOMStringList
 		const orgTypes = e.clipboardData.types;
-		let types = [];
-		if ( orgTypes.contains ) {
-			types = Array.from( orgTypes );
-		} else {
-			types = orgTypes;
-		}
+		const types = orgTypes.contains ? Array.from( orgTypes ) : orgTypes;
 		return ( Array.isArray( types ) && types.some( this.isGoogleDocType ) );
```

Comment:
Superfluous parens around the whole expression. Also, after the type cast on the previous line, do you ever expect `types` not to be an array? (I'm fine with playing it safe and keeping the `isArray` check.)